sim_world/config/wrist_controllers.yaml

```diff
@@ -0,0 +1,140 @@
+
+#---------------------------------------------------------------------
+# Controllers v1.1
+#---------------------------------------------------------------------
+
+#---------------------------------------------------------------------
+# Publish all joint states -----------------------------------
+#---------------------------------------------------------------------
+
+# joint_state_controller:
+#   type: joint_state_controller/JointStateController
+#   publish_rate: 50
+#   pid: { p: 100.0, i: 0.01, d: 3.0 }
+
+#---------------------------------------------------------------------
+# Effort- Controllers ---------------------------------------
+#---------------------------------------------------------------------
+
+j_wrist_rotation_eff_controller:
+  type: effort_controllers/JointEffortController
+  joint: j_wrist_rotation
+  pid: { p: 100.0, i: 0.01, d: 3.0 }
+j_wrist_exfle_eff_controller:
+  type: effort_controllers/JointEffortController
+  joint: j_wrist_exfle
+  pid: { p: 100.0, i: 0.01, d: 3.0 }
+j_wrist_ulra_eff_controller:
+  type: effort_controllers/JointEffortController
+  joint: j_wrist_ulra
+  pid: { p: 100.0, i: 0.01, d: 3.0 }
+
+#---------------------------------------------------------------------
+# Effort-Position Controllers ---------------------------------------
```

Comment:
Why do the controllers not have identical PID gains? Is it intentional?

Reply:
I just used the same gains as the finger controllers. If we find the need to retune them, we can easily do so.